e2e/cypress/integration/plugins/marketplace/marketplace_spec.js

```diff
@@ -321,5 +321,15 @@ describe('Plugin Marketplace', () => {
                 cy.get('#marketplaceTabs-pane-allPlugins').should('exist');
             });
         });
+
+        // This test is disabled until the marketplace instance with support for labels is deployed.
+        // This tests need to get updated when the labels send down from the Plugin Marketplace change.
+        xit('should show OFFICIAL label for github plugin', () => {
+            // # Scroll to GitHub plugin
+            cy.get('#marketplace-plugin-github').scrollIntoView().should('be.visible');
+
+            cy.get('#marketplace-plugin-github').find('.tag').should('be.visible').and('to.contain', 'OFFICIAL').trigger('mouseover');
+            cy.get('div.tooltip-inner').should('be.visible').and('contain', 'This plugin is maintained by Mattermost');
```

Comment:
Please let me know if these tests are fine

Reply:
Looks good to me. The only thing is I'd add more comments to the last two lines that it's finding the 'OFFICIAL' tag and verifying the tooltip behavior.

Reply:
It might also be nice to have the tests for the "All Plugins" and "Installed" tab in this section.

Reply:
I've added comments to the steps. Thanks for the suggestion :+1: 

There are already changes for two tabs at the end of the file.